adsb/adsb-ui/build.rs

```diff
@@ -0,0 +1,47 @@
+use std::{
+    io::Write,
+    process::{Command, Output},
+};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    println!("cargo:rerun-if-changed=./ui");
```

Comment:
I have no idea to be honest, i stole all this from the todo app and massaged it until it worked.  I have embedded Go working now, so maybe I can switch to that since I understand it.  Or @thomastaylor312 can provide some insight?